data/tab-category.json

```diff
@@ -97,6 +97,13 @@
     "link": null
 },
 {
+    "tab": "impl",
+    "category": "guidelines",
+    "labels": [],
```

Comment:
Do you want to filter the issues at all? Or just have all the issues shown?

Reply:
You're right, it would be more useful to show a more curated list. I added a commit to filter them.